app/controllers/api/userController.js

```diff
@@ -10,17 +10,21 @@ router.post(
         req.checkBody('txtIdUser').trim();
         req.checkBody('txtUsername').trim().notEmpty();
         req.checkBody('txtPassword').trim().notEmpty();
-        req.checkBody('txtEmail').trim();
+        req.checkBody('txtEmail').trim().notEmpty();
 
-        const [username, plainPassword, id, email]=[req.body.txtUsername, req.body.txtPassword, req.body.txtIdUser, req.body.txtEmail];
+        const [username, plainPassword, id, email] = [req.body.txtUsername, req.body.txtPassword, req.body.txtIdUser, req.body.txtEmail];
 
         let errors = req.validationErrors();
-
         if (errors) {
+            let message;
+            if (errors[0].param !== undefined && errors[0].param != null && errors[0].param.length > 0)
+                message = res.__('api.user.fields.empty.with.field') + errors[0].param;
+            else
+                res.__('api.user.fields.empty');
             res.send({
                 success: false,
-                message: res.__('api.user.fields.empty')
-            });
+                message
```

Comment:
mesagge: message?

Reply:
This syntax is valid and it gives you a JSON like `{success: false, message: '$VALUE$'}`, but for comprehension, I've changed it for the common syntax.